notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -61,7 +61,7 @@
     "\"git+https://github.com/huggingface/optimum-intel.git\"\\\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    from functools import wraps
</pre><p>with increasing number of supported models this code section become more and more large, I think maybe it is time for moving it in separated file?  it also will allow do not store model to converter mapping in notebook and directly map on configuration level</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
done